chaosaws/msk/probes.py

```diff
@@ -0,0 +1,43 @@
+from typing import List
+
+from chaoslib.types import Configuration, Secrets
+from chaoslib.exceptions import FailedActivity
+from chaosaws import aws_client, get_logger
```

Comment:
please add an extra blank line right above, between third party dependencies (chaoslib) and the local package (chaosaws)